lib/agent0/tests/econ_tests.py

```diff
@@ -0,0 +1,28 @@
+"""Tests of economic intuition."""
+
+import pytest
+
+from fixedpointmath import FixedPoint
+
+from agent0.hyperdrive.interactive import InteractiveHyperdrive
+from agent0.hyperdrive.interactive.chain import Chain
+
+YEAR_IN_SECONDS = 31_536_000
+
+
+@pytest.mark.anvil
+def test_asymmetry(chain: Chain):
+    """Does in equal out?"""
```

Comment:
can you add a more informative description for why this test is relevant or interesting?